README.md

```diff
@@ -1 +1,26 @@
 # Community Operators
+
+A collection of operators accessible from Operator Hub
+
+Operators stored here are pushed to quay.io's AppRegistry using [`operator-courier`](https://github.com/operator-framework/operator-courier)
+
+## Adding your operator
+
+To add your operator to this repo, submit a PR with your operator resources in a new directory named after your operator beneath the `rh-operators/` directory:
```

Comment:
Should we call this folder `rh-operators`? I believe outside teams will be using this `README` as well.

Reply:
@aravindhp mentioned that we should have a directory for `rh-operators` and a separate directory for `community` to keep them organized. I'm not sure which category we should be referring others to use since it depends.

Reply:
We are not ready to accept external community operators yet. So for now having `rh-operators` directory is fine. 

@awgreene the community operators you created would go into the `community` directory.

Reply:
I had missed that this was under the `okd operator` section. Thank you.